pysep/job.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "pysep-1.0",
+  "description": "Containerized pysep jupyter nb",
+  "appId": "pysep",
+  "appVersion": "1.0",
+  "archiveOnAppError": false,
+  "nodeCount": 4,
+  "coresPerNode": 1,
+  "jobType": "BATCH",
+  "memoryMB": 4000,
+  "maxMinutes": 45,
+  "isMpi": false,
+  "fileInputs": [],
+  "fileInputArrays": [],
+  "parameterSet": {
+    "appArgs": [],
+    "containerArgs": [],
+    "schedulerOptions": [
+      {
+        "name": "allocation",
+        "description": "my allocation",
+        "include": true,
+        "arg": "-A TACC-ACI-CIC"
+      }
```

Comment:
Set "arg" prop to be empty string
Also maybe, we should git rid of it? Up to you